content/dstu2/authorization.md

```diff
@@ -93,45 +89,102 @@ The [bearer access token][7] returned from the authorization server is what you
+
+##### Error URNs for the Authorization Server #####
+* **urn:cerner:error:authorization-server:oauth2:grant:invalid-request** - The authorization request was syntactically invalid.
+* **urn:cerner:error:authorization-server:oauth2:grant:unknown-client** - The client application is not registered.
+* **urn:cerner:error:authorization-server:oauth2:grant:invalid-redirect-uri** - The requested redirect URI does not match the one registered for the client application.
+* **urn:cerner:error:authorization-server:oauth2:grant:invalid-redirect-uri-trailing-slash** - The requested redirect URI does not match the one registered for the client application because of a trailing slash at the end of one of the URIs.
+* **urn:cerner:error:authorization-server:oauth2:grant:redirect-uri-not-absolute** - The requested or registered redirect URI is not absolute.
+* **urn:cerner:error:authorization-server:oauth2:grant:redirect-uri-not-registered** - The requested or registered redirect URI is not registered.
+* **urn:cerner:error:authorization-server:oauth2:grant:unknown-tenant** - The client redirect URI was known and valid, but the tenant provided was unknown.
+* **urn:cerner:error:authorization-server:oauth2:grant:tenant-not-configured** - The tenant id was known and valid, but the identity service provider id for the provided persona was not.
+* **urn:cerner:error:authorization-server:oauth2:grant:unauthorized-client-for-tenant** - The client redirect URI was known and valid, but is not authorized for use with the specified tenant.
+* **urn:cerner:error:authorization-server:oauth2:grant:server-error** - The client redirect URI was known and valid, but other fatal errors occurred during processing.
+* **urn:cerner:error:authorization-server:oauth2:grant:unsupported-response-type** - A response type other than "code" was requested.
+* **urn:cerner:error:authorization-server:oauth2:grant:csrf-security-failure** - The CSRF token used in confirming the user's approval was invalid; we will immediately notify the client app that we are unable to satisfy the request.
+* **urn:cerner:error:authorization-server:oauth2:grant:denied-by-server** - The server denied the grant because the client is not authorized for any of the scopes it requested.
+* **urn:cerner:error:authorization-server:oauth2:grant:denied-by-user** - The user denied the grant, either directly, or by choosing to cancel during the authentication process.
+* **urn:cerner:error:authorization-server:oauth2:grant:invalid-patient-id** - The patient id does not exist or the user does not have access to the selected patient id's records.
+* **urn:cerner:error:authorization-server:oauth2:grant:authorized-representative-server-error** - A server error was received from the authorized representative service.
+
```

Comment:
Looks like these blank lines aren't rendering anything either, but I can't figure out how to get line breaks in the middle of lists